spec/controllers/api_controller_spec.rb

```diff
@@ -1,12 +1,20 @@
 require 'rails_helper'
+require 'json'
 
 RSpec.describe ApiController, :type => :controller do
   before(:each) { login_admin }
 
   it 'routes to the all action on default' do
-    expect(:get => "/api").to route_to(
+    expect(:get => api_export_all_path).to route_to(
       :controller => "api",
-      :action => "all",
+      :action => "export_all",
     )
   end
+
+  it 'returns suppoprted formats' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.